web/src/PickNeighbourhoodMode.svelte

```diff
@@ -125,15 +126,22 @@
   function fillOpacity(
     selectedPrioritization: Prioritization,
   ): DataDrivenPropertyValueSpecification<number> {
-    return {
-      none: hoverStateFilter(0.3, 0.5),
+    let highlightedOpacity = 0.5;
+    let styles: Record<string, DataDrivenPropertyValueSpecification<number>> = {
+      none: [
+        "case",
+        ["==", ["feature-state", "highlight"], "yes"],
+        highlightedOpacity,
+        hoverStateFilter(0.0, highlightedOpacity),
```

Comment:
I like a very slight fill -- 0.2 here feels good:
![image](https://github.com/user-attachments/assets/19e50444-2ecb-41d0-a8c0-416ae4d96477)
Yellow looks OK, but blue would look fine too. But yeah, I think somehow styling as a 2D area is still important